### PR TITLE
PID: Move from large tables to split tables

### PR DESCRIPTION
**Pull request by njacazio**
- Improve handler of TOF parametrization
- Add dedicated PID directory for PID tasks
- Add TOF beta task with dedicated QA
- Move from TOF full table to TOF split tables
- Move from TPC full table to TPC split tables
- Add QA tasks for split PID tables
- The PID split task is the PID default task

### CODE REVIEW

**Comment by njacazio** [comment]
This PR is a subset of https://github.com/AliceO2Group/AliceO2/pull/5784 so as to factorize the transparent changes.
It also depends on the outcome of https://github.com/AliceO2Group/AliceO2/pull/5914, I will mark this for merge once this is merged.

**Comment by njacazio** [comment]
Before merging I would wait for a green light from @iarsene but the review can already start.

**Review by iarsene — APPROVED** [review]

**Comment by iarsene** [comment]
Hi @njacazio,
The changes are fine from my side. The fullCI crash is unrelated.

File: Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h
Comment (njacazio) [inline Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h:40]: @vkucera this should be transparent from the point of view of the O2, concerning the Run3 validation sw the flags corresponding to the mass hypotheses should be turned on after this is merged.
e.g. `o2-analysis-pid-tof --pid-pi 1 --pid-pr 1` and so on, let me know if you have any issue!

File: Analysis/Tasks/PWGLF/NucleiSpectraTask.cxx
Comment (njacazio) [inline Analysis/Tasks/PWGLF/NucleiSpectraTask.cxx:64]: @lbariogl I now move from the full table to the split one by default, this is transparent for you, on the pidTOF and pidTPC side, i.e. `o2-analysis-pid-tof --pid-he 1`. This is to let you know.

File: Analysis/Tasks/SkimmingTutorials/spectraNucleiProvider.cxx
Comment (njacazio) [inline Analysis/Tasks/SkimmingTutorials/spectraNucleiProvider.cxx:52]: @nzardosh I now move from the full table to the split one by default, this is transparent for you, on the pidTOF and pidTPC side, e.g. `o2-analysis-pid-tof --pid-pi 1 --pid-pr 1` This is to let you know, maybe there are even more particle species that can be removed in the future.
The same applies to the other tasks.

File: Analysis/Tasks/PWGDQ/filterPP.cxx
Comment (njacazio) [inline Analysis/Tasks/PWGDQ/filterPP.cxx:63]: @iarsene I now move from the full table to the split one by default, this is transparent for you, on the pidTOF and pidTPC side, i.e. `o2-analysis-pid-tof --pid-el 1` and so on. This is to let you know.

**Review by njacazio — COMMENTED** [review]
Hi @vkucera  @iarsene  @lbariogl @nzardosh I would like to push some changes to your code, this is to let you know
If you could acknowledge the changes we can merge it, thanks!

File: Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h
Comment (vkucera) [reply Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h:40]: Fine with me.

**Review by vkucera — APPROVED** [review]

**Comment by njacazio** [comment]
Green light obtained!

**Pull request merged**
